### PR TITLE
Add CardMarket

### PR DESCRIPTION
**Pull request by ColinHebert**
cardmarket.com [rank 9,028](https://www.alexa.com/siteinfo/cardmarket.com) supports TOTP, add details about it.

### CODE REVIEW

File: img/c/cardmarket.com.svg [hunk 1]
```diff
@@ -0,0 +1,4 @@
...
+  <path d="M81 57l30.73 138.7c.73 2.56 2.3 6.4 6.14 8.92 4.8 3.18 8.43 3.03 9.23 3h10.88l-18.9 7s-2.2.86-4.6-.1c-2.22-.9-3.33-2.93-3.33-2.93L59.92 71.56s-.9-1.88.22-4.78c1.03-2.6 3.35-3.4 3.35-3.4L81 57zm29.65-25.18v121.82L87.43 43.16s-.5-2.5.94-4.65c1.8-2.74 4.56-3.22 4.56-3.22l17.72-3.47zm59.52 100.48c-1.88 0-3.66-.7-5-2l-29.97-28.8c-1.38-1.33-2.16-3.12-2.18-5.04a7.02 7.02 0 0 1 2.06-5.07l44.67-44.95a.95.95 0 0 0 .28-.68.92.92 0 0 0-.3-.68l-11.6-11.15c-2.45-2.35-1.93-4.53-1.6-5.36s1.47-2.76 4.86-2.76l61.9.04a7.17 7.17 0 0 1 7.16 7.16l-.04 60.7c0 3.57-2.26 5.2-4.35 5.2-.9 0-2.3-.3-3.8-1.73l-11.6-11.14c-.27-.26-.58-.3-.76-.3s-.5.04-.78.3l-43.82 44.1c-1.34 1.38-3.16 2.13-5.1 2.13m27.57 62.8l-67.22-.06a7.47 7.47 0 0 1-5.29-2.19c-1.42-1.42-2.2-3.3-2.2-5.3l.05-66.27c0-2.65 1.27-4.76 3.4-5.63s4.5-.27 6.36 1.6l24.1 24.3c1.73 1.74 1.72 4.56-.03 6.3-1.75 1.72-4.56 1.7-6.3-.03L132 129.02l-.04 57.08 57.8.05-8.17-7.88c-1.44-1.4-2.25-3.27-2.28-5.28s.74-3.9 2.16-5.32l50.6-51.14c1.73-1.75 4.55-1.76 6.3-.04 1.75 1.73 1.76 4.55.04 6.3l-49.58 50.1 12.88 12.42c1.92 1.85 2.55 4.23 1.68 6.36-.86 2.13-2.96 3.4-5.63 3.4" fill="#012169"/>
+</svg>
```
Comment (Carlgo11) [inline img/c/cardmarket.com.svg]:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 180.87 189.16"><path fill="#012169" d="m21.42 31.19 30.73 138.7c.73 2.56 2.3 6.4 6.14 8.92 4.8 3.18 8.43 3.03 9.23 3H78.4l-18.9 7s-2.2.86-4.6-.1a7.28 7.28 0 0 1-3.33-2.93L.34 45.75s-.9-1.88.22-4.78a6.1 6.1 0 0 1 3.35-3.4zM51.07 6.01v121.82L27.85 17.35s-.5-2.5.94-4.65c1.8-2.74 4.56-3.22 4.56-3.22zm59.52 100.48c-1.88 0-3.66-.7-5-2l-29.97-28.8c-2.67-3.08-2.77-7.38-.12-10.11l44.67-44.95a.95.95 0 0 0-.02-1.36l-11.6-11.15c-2.45-2.35-1.93-4.53-1.6-5.36.33-.83 1.47-2.76 4.86-2.76l61.9.04c3.95 0 7.16 3.2 7.16 7.16l-.04 60.7c0 3.57-2.26 5.2-4.35 5.2-.9 0-2.3-.3-3.8-1.73l-11.6-11.14c-.43-.4-1.1-.4-1.54 0l-43.82 44.1a7.05 7.05 0 0 1-5.1 2.13m27.57 62.8-67.22-.06a7.47 7.47 0 0 1-7.49-7.49l.05-66.27c0-2.65 1.27-4.76 3.4-5.63 2.13-.87 4.5-.27 6.36 1.6l24.1 24.3c4.18 4.22-2.15 10.49-6.33 6.27l-18.64-18.77-.04 57.08 57.8.05-8.17-7.88a7.46 7.46 0 0 1-.12-10.6l50.6-51.14c4.17-3.84 10.13 2.04 6.34 6.26l-49.58 50.1 12.88 12.42c1.92 1.85 2.55 4.23 1.68 6.36-.86 2.13-2.96 3.4-5.63 3.4"/></svg>
```

File: entries/c/cardmarket.com.json [hunk 2]
```diff
@@ -0,0 +1,12 @@
+{
+  "CardMarket": {
+    "domain": "cardmarket.com",
+    "recovery": "https://cardmarket.com/en/Login/LostDevice",
+    "tfa": [
+      "totp"
+    ],
+    "keywords": [
+      "gaming"
```
Comment (Carlgo11) [inline entries/c/cardmarket.com.json]: I think the retail category would be a better fit as, unlike Steam & Origin that are retail platforms and game launchers, CardMarket is only a retail platform.

```suggestion
      "retail"
```

**Review by Carlgo11 — CHANGES_REQUESTED** [review]
I see that you haven't add any documentation link. Could you maybe find one that describes how to enable 2FA on the site?

If no documentation is available or you can't find it, could you please add some screenshots of the 2FA setup process and/or login process?
__Remember to blur or crop out any personal information linked to you.__

You can attach the screenshots here in a comment, like this:
screenshot

File: entries/c/cardmarket.com.json (hunk 2 above)
Comment (ColinHebert) [reply entries/c/cardmarket.com.json]: That works for me

**Comment by ColinHebert** [comment]
Absolutely, here's a screenshot of the setup page
Screen Shot 2022-01-02 at 00 27 15

**Comment by Carlgo11** [comment]
That's a large screenshot! What monitor is that if you don't mind me asking :eyes: 

Since the text doesn't mention how the 2FA is implemented, are the codes required upon sign in or just when performing certain actions such as buying and selling?

**Comment by ColinHebert** [comment]
>That's a large screenshot! What monitor is that if you don't mind me asking 👀

The joys of MacOS taking screenshots on retina display at crazy resolutions.

> are the codes required upon sign in or just when performing certain actions such as buying and selling?

Actually, this is customisable. here are the defaults
Screen Shot 2022-01-02 at 08 40 45

**Review by Carlgo11 — APPROVED** [review]
Thanks for all the info and screenshots! Approving your PR now.

**Review by phallobst — APPROVED** [review]

**Pull request merged**